### PR TITLE
Markup content to match deprecation in BCD

### PR DESCRIPTION
**Pull request by hamishwillee**
This adds deprecation and obsolete markup to a number of pages in order to match the BCD.

### CODE REVIEW

File: files/en-us/web/api/vrdisplay/isconnected/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: VRdisplay.isConnected
+title: VRDisplay.isConnected
```
Comment (hamishwillee) [inline files/en-us/web/api/vrdisplay/isconnected/index.html:2]: Note, old title is typo

File: files/en-us/web/api/globaleventhandlers/index.html [hunk 2]
```diff
@@ -69,7 +69,7 @@ <h3 id="Event_handlers">Event handlers</h3>
  <dd>Is an {{domxref("EventHandler")}} representing the code to be called when the {{event("dragend")}} event is raised.</dd>
  <dt>{{domxref("GlobalEventHandlers.ondragenter")}}</dt>
  <dd>Is an {{domxref("EventHandler")}} representing the code to be called when the {{event("dragenter")}} event is raised.</dd>
- <dt>{{domxref("GlobalEventHandlers.ondragexit")}}</dt>
+ <dt>{{domxref("GlobalEventHandlers.ondragexit")}} {{Deprecated_Inline}}</dt>
```
Comment (hamishwillee) [inline files/en-us/web/api/globaleventhandlers/index.html:72]: Note, this actually links to ondragleave, which is NOT deprecated.

**Comment by Rumyra** [comment]
@hamishwillee as this is the longest commit 😆 I've browsed the files and it looks fine. I went ahead and resolved a couple of conflicts - let me know if this looks ok to you and I'll merge 👍

**Comment by hamishwillee** [comment]
Hi @Rumyra 
Yes, please do merge this - if I could have done so myself I would have :-). I'll probably do a further fix to remove the "obsolete" markup. Apparently we only use deprecated now. But that's for another day.

**Review by chrisdavidmills — APPROVED** [review]

**Comment by chrisdavidmills** [comment]
Thought I'd just get this done as I was passing by...

**Pull request merged**